main.go

```diff
@@ -99,24 +99,18 @@ func createConfigsModelFromEnvs() ConfigsModel {
 	}
 }
 
+func strip(str string) string {
+	str = strings.TrimPrefix(str, "\n")
```

Comment:
TrimPrefix and TrimSuffix removes only one `\n`, but as it is a custom input, here can be more than one `\n`s